src/test/java/de/caritas/cob/userservice/api/adapters/web/controller/UserControllerSessionE2EIT.java

```diff
@@ -730,29 +683,11 @@ void getSessionsForGroupOrFeedbackGroupIdsShouldContainConsultantOfUserSession()
         .andExpect(jsonPath("sessions", hasSize(1)));
   }
 
-  @Test
-  @WithMockUser(authorities = {AuthorityValue.USER_DEFAULT})
-  void getSessionsForGroupOrFeedbackGroupIdsShouldBeForbiddenIfUserDoesNotParticipateInSession()
-      throws Exception {
-    givenAUserWithSessions();
-    givenNoRocketChatSubscriptionUpdates();
-    givenNoRocketChatRoomUpdates();
-
-    mockMvc
-        .perform(
-            get("/users/sessions/room?rcGroupIds=4SPkApB8So88c7tQ3")
-                .cookie(CSRF_COOKIE)
-                .header(CSRF_HEADER, CSRF_VALUE)
-                .header(RC_TOKEN_HEADER_PARAMETER_NAME, RC_TOKEN)
-                .accept(MediaType.APPLICATION_JSON))
-        .andExpect(status().isForbidden());
-  }
-
   @ParameterizedTest
   @WithMockUser(authorities = {AuthorityValue.CONSULTANT_DEFAULT})
   @ValueSource(strings = {"QBv2xym9qQ2DoAxkR", "doesNotExist", "mzAdWzQEobJ2PkoxP"})
   void
-      getSessionsForGroupOrFeedbackGroupIdsShouldBeNoContentIfConsultantDoesNotParticipateInSessionOrNoSessionsFoundForIdsOrNewEnquiriesForConsultantsNotInAgency(
+      getSessionsForGroupIdsShouldBeNoContentIfConsultantDoesNotParticipateInSessionOrNoSessionsFoundForIdsOrNewEnquiriesForConsultantsNotInAgency(
```

Comment:
hmm is there no need to change the setup?
Was that test tests two things?

Reply:
That test tests calling the api "/users/sessions/room?rcGroupIds=" 3 times with the groupids "QBv2xym9qQ2DoAxkR", "doesNotExist", "mzAdWzQEobJ2PkoxP". Each of them testing 3 different cases of a no content response by the api. I think there is no need to change the setup.

Reply:
I just asked because of the changed name